src/Resources/web/app_dev.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2017 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+use Contao\ManagerBundle\ContaoManager\Plugin as ManagerBundlePlugin;
+use Contao\ManagerBundle\HttpKernel\ContaoKernel;
+use Doctrine\Common\Annotations\AnnotationRegistry;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Dotenv\Dotenv;
+use Symfony\Component\HttpFoundation\Request;
+
+// The check is to ensure we don't use .env in production
```

Comment:
indeed no bundle should directly access env vars, but use the DI containers parameters.
however there is no parameters.yml in symfony flex anymore, since it is expected that all deployment target dependant config is passed through env vars and then passed through to the containers parameters. how this applies to contao settings like "allowed file types" idk, because that is clearly not a config you would pass through env vars. imho the cleanest way, would be to use symfonies abilities to stack semantic config and allow to a contao admin to edit the semantic config directly (and not through container parameters).